sources/lib/reasoners/intervalCalculus.ml

```diff
@@ -1663,9 +1740,10 @@ let add =
     try
       let env = {env with new_uf} in
       if is_num r then
+        let env, eqs = add_used_by r env in
```

Comment:
I think in `fun env new_uf r _ ->` the `_` is the term (expr) you'd like to pass to `add_used_by` to avoid `X.term_extract x` is the latter function